app/Resources/views/admin/layout.html.twig

```diff
@@ -7,6 +7,12 @@
 #}
 {% extends 'base.html.twig' %}
 
+{#
+    Import macros used for displaying flash messages.
+    See http://twig.sensiolabs.org/doc/tags/import.html
+#}
+{% import 'macro/messages.html.twig' as helper %}
```

Comment:
same comment that in the original PR. Importing macros in the parent to use them in the child is not a supported way to use macros and will be removed in Twig 2.x